src/nvim/api/vim.c

```diff
@@ -33,6 +34,22 @@
 # include "api/vim.c.generated.h"
 #endif
 
+/// @see foldCreate
```

Comment:
would be better to move the doc here (and put `@see nvim_fold_foo` in the old place), since these doxygen comments get generated to `:help api-funcs`